floodfill_selection/volume_selection.py

```diff
@@ -226,26 +230,29 @@ def mark_selected(self):
 
     def get_max_pos(self):
         # Ray intersection on the CPU to highlight the selected point(s)
-        data = self.tr.map(self.pos_data)[:, :2]  # Map coordinates
-        print('data after tr.map', data)
+        data = self.transform(self.pos_data)  # Map coordinates
         m1 = data > (self.selection_origin - 4)
         m2 = data < (self.selection_origin + 4)
         max_value = 0.
         max_pos = None
         pick_selected = np.argwhere(m1[:,0] & m1[:,1] & m2[:,0] & m2[:,1])
         for item in pick_selected:
-            index = np.unravel_index(item, self.vol_data.shape)
+            index = tuple(self.pos_data[item].flatten())
```

Comment:
Ok, so I cannot figure out why, but unravel_index gives the wrong answer. But we can just get the index from ``pos_data``. I think this was the issue for why the incorrect position was being selected (at least it was for me)